packages/tasks/src/dataset-libraries.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Elements configurable by a dataset library.
+ */
+export interface DatasetLibraryUiElement {
+	/**
+	 * Pretty name of the library.
+	 * displayed (in tags?, and) on the main
+	 * call-to-action button on the dataset page.
+	 */
+	prettyLabel: string;
+	/**
+	 * Repo name of the library's (usually on GitHub) code repo
+	 */
+	repoName:    string;
+	/**
+	 * URL to library's (usually on GitHub) code repo
+	 */
+	repoUrl:     string;
+	/**
+	 * URL to library's docs
+	 */
+	docsUrl?:    string;
+	/**
+	 * URL to library's docs in the Hugging Face documentation
+	 */
+	hfDocsUrl?:  string;
```

Comment:
imo `docsUrl` can always point to hub docs, which in turn point to the library docs (like for models)

Reply:
that way we have one less URL to define

Reply:
done